.github/workflows/check-pr-title.yml

```diff
@@ -0,0 +1,16 @@
+name: Check PR Title
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+jobs:
+  check-pr-title:
+    name: Check PR Title
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR Title
+        uses: rudderlabs/github-action-check-pr-title@v1.0.7
```

Comment:
Is it possible to club `check-pr-title.yaml` & `checks-semantic-pr.yml` by having validations in config file here ?
Ref: https://github.com/marketplace/actions/pr-title-checker#usage

Reply:
Yes, one of them is redundant. I will fix it.

Reply:
Done.